src/prepare.ts

```diff
@@ -30,8 +33,9 @@ export const prepare = async ({ cabalFile }: PluginConfig, { nextRelease, logger
   }
 
   const fullCabalPath = resolve("./", cabalFileName);
+  const fullVersion = versionPrefix ? versionPrefix + version : version;
```

Comment:
You can avoid the ternary if you add a default value to `versionPrefix` and then just join the strings:
```ts
export const prepare = async (
  { cabalFile, versionPrefix = "" }: PluginConfig, // defaults to empty string
  { nextRelease, logger }: PrepareContext,
): Promise<void> => {
  // ...rest of code

  const fullVersion = `${versionPrefix}${version}`;
}
```